### DISCUSSION TITLE
Guide to use lava_exchange.py

### DISCUSSION THREAD

**Discussion by CloudyDory** [fangwei123456/SpikingFlow#339] [Q&A] [answered]
Hi, is there a documentation about how to use "lava_exchange.py"? I have a trained SNN in spikingjelly, can I use this file to convert the network to run under Lava?

**Comment by fangwei123456** [comment]
Hi, thanks for your suggestion. We will add a tutorial in this week.

**Comment by fangwei123456** [comment, marked as answer]
Hi, the tutorial is added:

https://spikingjelly.readthedocs.io/zh_CN/latest/activation_based_en/lava_exchange.html

**Reply by CloudyDory** [reply]
Thank you very much! Really appreciate your work!
